presentations/index.md

```diff
@@ -128,7 +141,8 @@ Red Hat Summit 2014, San Francisco, California, USA
 ### April 14-17, 2014
 
 -   [Red Hat Storage Server Administration Deep
```

Comment:
I'm not sure if this belongs under GlusterCommunity account as it has RHGS content.